docs/events.md

```diff
@@ -0,0 +1,88 @@
+# Player Events
+A string representing an event within the player.
+
+## Overview
+
+The player events are consisted of two event types:
+
+ 1. HTML5 events - various events sent by the browser when handling media embedded using the `<video>` element.
+ 2. Kaltura player custom events - special events indicating changes in the state of the player that does not exist in the html5 video event list and are related to the integral behavior of the player. Such as ads, fullscreen and subtitles events.
```

Comment:
Kaltura player custom events - special events that indicate changes in the state of the player, which do not exist in the html5 video event list and are related to the integral behavior of the player, such as ads, fullscreen and subtitles events.